lib/rbs_rails/active_record.rb

```diff
@@ -517,7 +518,39 @@ def clear_#{col.name}_change: () -> void
           sig
         end.join("\n")
         mod_sig << "\nend\n"
-        mod_sig << "include GeneratedAttributeMethods"
+        mod_sig << "include GeneratedAttributeMethods\n"
+        mod_sig
+      end
+
+      private def alias_columns
+        mod_sig = +"module GeneratedAliasAttributeMethods\n"
+        mod_sig << "include GeneratedAttributeMethods\n"
+        mod_sig << klass.attribute_aliases.map do |col|
+          sig = <<~EOS
+            alias #{col[0]} #{col[1]}
```

Comment:
During development, I noticed that each method was defined with the old attribute, even though the name defined in `alias_attribute` was used in the `attribute`.
Therefore, we thought it would be better to use `alias` and made the changes